assets/aml-benchmark/components/batch-resource-manager/spec.yaml

```diff
@@ -7,6 +7,9 @@ description: Resource Manager for batch inference.
 version: 0.0.1
 
 inputs:
+  wait_input:
+    type: uri_file
+    optional: True
```

Comment:
just curious, what is the use of this?

Reply:
I just add a fake input that the component can wait until the run is finished. Then it can execute deletion of the endpoiint

Reply:
How are we using this? This input is not even passed to the `main` script. Am I missing something?

Reply:
@iamrk04 , I believe this input will be connected to some output of inferencing component so that it runs after it has finished. It is marked as optional so that even if inferencing fails, this component can run. That is why it is "wait_input", just to make sure it runs __after__ inferencing is done.